src/libsync/account.cpp

```diff
@@ -85,6 +87,14 @@ void Account::setUser(const QString &user)
     _user = user;
 }
 
+QString Account::davFilesPath() const
+{
+    //TODO DO NOT HARCODE PATH, GET IT FROM THE SERVER!!!!
+    QString dfp("/remote.php/dav/files/");
```

Comment:
see https://github.com/owncloud/client/pull/5332

With that change, you don't need this change anymore and you can just use the davPath